packages/cursorless-org/package.json

```diff
@@ -15,8 +15,8 @@
   "dependencies": {
     "@cursorless/cheatsheet": "workspace:*",
     "eslint": "^8.38.0",
-    "eslint-config-next": "13.2.3",
-    "next": "13.2.3",
+    "eslint-config-next": "13.5.4",
```

Comment:
A question for later: can we move these ESLint and TypeScript deps out of the prod dependencies?

Reply:
Yeah was thinking the same thing when I bumped this. Presumably there's no reason for those to be prod dependencies